internal/workermanager/in_situ_worker.go

```diff
@@ -32,4 +35,62 @@ func (m *InSituWorker) EnqueueDeleteAssetJob(ctx context.Context, urn string) er
 	return nil
 }
 
+func (m *InSituWorker) EnqueueSyncAssetJob(ctx context.Context, service string) error {
```

Comment:
Whats the difference between this job and the new code block in the file - `internal/workermanager/discovery_worker.go`? they seem to have exactly same functionality.

Reply:
the one in discovery_worker will only run in compass worker by registering the job to jobs_queue. While this in_situ worker will run in compass API without queueing anything, and execute the sync immediately